libdokan/fs.go

```diff
@@ -151,7 +151,16 @@ func (f *FS) GetDiskFreeSpace(ctx context.Context) (freeSpace dokan.FreeSpace, e
 			FreeBytesAvailable:     dummyFreeSpace,
 		}, nil
 	}
-	defer func() { f.reportErr(ctx, libkbfs.ReadMode, err) }()
+	defer func() {
+		if err == nil {
+			f.log.CDebugf(ctx, "Request complete")
+		} else {
+			// Don't report the error (perhaps resulting in a user
+			// notification) since this method is mostly called by the
+			// OS and not because of a user action.
+			f.log.CDebugf(ctx, err.Error())
```

Comment:
The error is already logged [below](https://github.com/keybase/kbfs/pull/953/files#diff-3c07e38475555636a646b747704ace53R167). Perhaps remove one of them? But the previous logs twice too, so I don't feel very strong about this